frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx

```diff
@@ -25,9 +25,16 @@ type Props = NodeProps<TableNodeType>
 
 export const TableNode: FC<Props> = ({ data: { table } }) => {
   const { relationships } = useDBStructureStore()
+  const [isOpen, setIsOpen] = useState(false)
+
   return (
     <>
-      <DrawerRoot direction="right">
+      <DrawerRoot
+        direction="right"
+        dismissible={false}
+        open={isOpen}
+        onOpenChange={setIsOpen}
```

Comment:
I'm not sure, but when dismissible turn to false, set handler will be needed.
I feel this behavior is not correct 🤔 and would appreciate your review.

Reply:
It seems to be necessary if dismissible=false!